modules/get-started/pages/licensing/overview.adoc

```diff
@@ -0,0 +1,88 @@
+:description: Redpanda Self-Managed is free and source-available at the Redpanda GitHub repo. Additional features are included with an Enterprise license.
+:page-aliases: introduction:licenses.adoc, get-started:licenses.adoc
+
+Redpanda Self-Managed, Redpanda Console, and Redpanda Connect are available in either the Community Edition or the Enterprise Edition. Each product has its own binary, and each binary supports both the Community Edition and Enterprise Edition. Features in the Enterprise Edition (enterprise features) are only accessible with a valid license key.
+
+* Redpanda Community Edition is free and source-available on GitHub:
+** https://github.com/redpanda-data/redpanda[Redpanda Self-Managed^]
+** https://github.com/redpanda-data/console[Redpanda Console^]
+** https://github.com/redpanda-data/connect[Redpanda Connect^]
+* Redpanda Enterprise Edition requires a license key and includes additional features.
+
+[NOTE]
+====
+Redpanda Cloud is a managed deployment of Redpanda Enterprise Edition. To learn more about Redpanda Cloud, see the xref:redpanda-cloud:get-started:cloud-overview.adoc[].
+====
+
+== Community Edition
+
+Community Edition is licensed with the Redpanda https://github.com/redpanda-data/redpanda/blob/dev/licenses/bsl.md[Business Source License^] (BSL). These core features are free and source-available.
+
+[NOTE]
+====
+* Users cannot provide Redpanda as a commercial streaming or queuing service to others.
+* The BSL code converts to Apache 2.0 licensing four years after each code merge.
+====
+
+== Enterprise Edition
+
+Enterprise Edition is licensed with the https://github.com/redpanda-data/redpanda/blob/dev/licenses/rcl.md[Redpanda Community License^] (RCL). It includes the free features licensed under the Redpanda BSL, as well as enterprise features.
+
+=== License keys
+
+Enterprise features require an Enterprise Edition license key, sometimes referred to as Enterprise license, license key, or license. You can evaluate enterprise features with a free 30-day trial. To request a trial license key, to extend your trial period, or to purchase a license, contact https://redpanda.com/try-redpanda?section=enterprise-trial[Redpanda Sales^].
+
+[[self-managed]]
+=== Enterprise features in Redpanda Self-Managed
+
+The following enterprise features for Redpanda Self-Managed are activated with a valid Enterprise Edition license key:
+
+include::get-started:partial$licensing/enterprise-features.adoc[tag=self-managed]
+
+[[console]]
+=== Enterprise features in Redpanda Console
+
+The following enterprise features for Redpanda Console are activated with a valid Enterprise Edition license key:
+
+include::get-started:partial$licensing/enterprise-features.adoc[tag=console]
+
+[[connect]]
+=== Enterprise features in Redpanda Connect
+
+The following enterprise features for Redpanda Connect are activated with a valid Enterprise Edition license key:
+
+include::get-started:partial$licensing/enterprise-features.adoc[tag=connect]
```

Comment:
What happens with all other connectors/components that are marked as Enterprise?

Reply:
We should have a single place that lists them all. Maybe we should host that in a partial in the `rp-connect-docs` repo?

Reply:
We can make that a follow-up issue. For now, I'll update the list with all the current Enterprise connectors.